packages/harpguru-core/src/components/harp-cell/harp-cell.tsx

```diff
@@ -74,7 +74,10 @@ export const HarpCell = ({ yxCoord }: HarpCellProps): React.ReactElement => {
   )
 
   const accessibleContent = (
-    <LongPressGestureHandler onHandlerStateChange={handleLongPressStateChange}>
+    <LongPressGestureHandler
+      onHandlerStateChange={handleLongPressStateChange}
+      minDurationMs={100}
```

Comment:
This needs increasing.

Reply:
Increased to 500. When we have a method of communicating that the action exists to the user then we can maybe bring it down so if someone does do it by accident they understand what they've done.